### ISSUE TITLE
Add logo to header

### ISSUE THREAD

**Issue by CoralSilver** [SC5/sc5-styleguide#715]
Just started using the generator, and so far it is awesome. It would be great if you could make it a build option to inject a logo into the header.

**Comment by khrome83** [comment]
Can we use CSS to inject one?

**Comment by CoralSilver** [comment]
Should be able to, though I haven't tried yet.  I will try putting in my variable overrides file.

**Comment by kraftner** [comment]
This should work as an example:

    @mixin styleguide_custom_styles {
     .sg-title:before {
      display: block;
      position: relative;
      top: 15px;
      float: left;
      margin-right: 5px;
      background: url('http://example.com/example.png') $white;
      background-size: cover;
      width: 40px;
      height: 40px;
      content: '';
     }
    }

I think you could close this issue now.

**Comment by hurrtz** [comment]
I agree. Although a defined way to publish a logo in the header and/or in the footer would be nice, it is absolutely doable with plain CSS and pseudo styles.
